ocs_ci/ocs/constants.py

```diff
@@ -140,7 +141,9 @@
 ]
 must_gather_pod_label = "must-gather"
 
-OCS_WORKLOADS = "https://github.com/red-hat-storage/ocs-workloads"
+# AMQ
+AMQ_NAMESPACE = "myproject"
```

Comment:
could you give meaningful namespace. may be "amq_namespace" ?

Reply:
`myproject` is default namespace defined in all the amq yamls, so kept the same namespace as default one